app/Core/StatusManager.js

```diff
@@ -91,12 +243,37 @@ StatusManager.prototype.newStatus = function(status) {
     }
 
     // Fire status event
-    this.eventHandler.emit('status', status);
+    this.eventHandler.emit('status', this.buildStatus(
+        status.project,
+        status.branch,
+        'api',
+        status.status,
+        'status'
+    ));
 
     return true;
 };
 
 /**
+ * Build a status object for the status modules
+ *
+ * @param {string} project
+ * @param {string} branch
+ * @param {string} source
+ * @param {string} status
+ * @param {string} name
+ */
+StatusManager.prototype.buildStatus = function(project, branch, source, status, name) {
+    return {
+        project: project,
+        branch: branch,
+        source: source,
+        status: status,
+        name: name
```

Comment:
Totally missing `type` in here. Should be filled with the stage in case of jobs.